dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderFactoryImpl.java

```diff
@@ -1208,12 +1223,16 @@ protected Identifier createIdentifierForFolder(Folder folder, String parentPath)
 
 	@Override
 	protected void save(Folder folderInode) throws DotDataException {
+		validateFolderName(folderInode);
+
 		HibernateUtil.getSession().clear();
 		HibernateUtil.saveOrUpdate(folderInode);
 	}
 
 	@Override
 	protected void save(Folder folderInode, String existingId) throws DotDataException {
```

Comment:
why do we have two save methods?,I think this method should call the other one

Reply:
This is old code, not part of the change. I will check it out anyways to see if it can be refactored

Reply:
I don't see a way to refactor in an easy/safe way. Leaving it like it is.